src/devices/uFire_ISE/UFireIse.cs

```diff
@@ -0,0 +1,388 @@
+        {
+            return new ElectricPotential(Read_register((byte)Register.ISE_CALIBRATE_READLOW_REGISTER), UnitsNet.Units.ElectricPotentialUnit.Millivolt);
+        }
+
+        /// <summary>
+        ///  Resets all the stored calibration information
+        /// </summary>
+        public void Reset()
+        {
+            Write_register((byte)Register.ISE_CALIBRATE_SINGLE_REGISTER, null);
+            Write_register((byte)Register.ISE_CALIBRATE_REFHIGH_REGISTER, null);
+            Write_register((byte)Register.ISE_CALIBRATE_REFLOW_REGISTER, null);
+            Write_register((byte)Register.ISE_CALIBRATE_READHIGH_REGISTER, null);
+            Write_register((byte)Register.ISE_CALIBRATE_READLOW_REGISTER, null);
+        }
+
+        /// <summary>
+        /// Sets all the values, in mV, for dual point calibration and saves them in the devices's EEPROM
+        /// </summary>
+        /// <param name="refLow">the reference low point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="refHigh">the reference high point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="readLow">the measured low point. Range: -1024 mV to 1024 mV</param>
+        /// <param name="readHigh">the measured high point. Range: -1024 mV to 1024 mV</param>
+        public void SetDualPointCalibration(ElectricPotential refLow, ElectricPotential refHigh, ElectricPotential readLow, ElectricPotential readHigh)
+        {
+            if (refLow.Millivolts >= -1024 && refLow.Millivolts <= 1024 &&
+                refHigh.Millivolts >= -1024 && refHigh.Millivolts <= 1024 &&
+                readLow.Millivolts >= -1024 && readLow.Millivolts <= 1024 &&
+                readHigh.Millivolts >= -1024 && readHigh.Millivolts <= 1024)
+            {
+                Write_register((byte)Register.ISE_CALIBRATE_REFLOW_REGISTER, Convert.ToSingle(refLow.Millivolts));
+                Write_register((byte)Register.ISE_CALIBRATE_REFHIGH_REGISTER, Convert.ToSingle(refHigh));
+                Write_register((byte)Register.ISE_CALIBRATE_READLOW_REGISTER, Convert.ToSingle(readLow));
+                Write_register((byte)Register.ISE_CALIBRATE_READHIGH_REGISTER, Convert.ToSingle(readHigh));
+            }
+        }
+
+        /// <summary>
+        /// Changes the default I2C address
+        /// </summary>
+        /// <param name="i2cAddress">The new I2C address</param>
+        public void SetI2CAddress(byte i2cAddress)
+        {
+            if (i2cAddress >= 1 && i2cAddress <= 127)
+            {
+                Write_register((byte)Register.ISE_SOLUTION_REGISTER, Convert.ToByte(i2cAddress));
+                Send_command((byte)Command.ISE_I2C);
+            }
+        }
+
+        /// <summary>
+        /// Low level reads of EEPROM
+        /// </summary>
+        /// <param name="address">Address to read</param>
+        /// <returns>Low level data from EEPROM</returns>
+        public float ReadEEPROM(float address)
```

Comment:
So why do you keep a float if the value is a byte address? 
if you use ```public float ReadEeprom(byte address)``` it makes it a bit easier.
Also consider throwing an Argument Exception then is address == 0xFF
And please consider using the correct naming convention so Eeprom rather than EEPROM when using for function, properties, variables